.github/workflows/testworkflow.yml

```diff
@@ -0,0 +1,60 @@
+name: testworkflow
```

Comment:
maybe `verify-results` here and `verify-results.yml` as filename would be more descriptive. and jobname below could be `verify`

Reply:
Ok, `testworkflow` has been replaced by `verify results`

Reply:
nice